src/App.vue

```diff
@@ -12,6 +12,7 @@
       <main>
         <router-view/>
       </main>
+      <long-button colorClass="bg-primary-dark text-gray-light" text="Provide Feedback" />
```

Comment:
Maybe have the button in the <main> for screen readers. It is part of the main content.